lib/kits/incito-publication/controls.js

```diff
@@ -16,14 +15,14 @@ export default class Controls {
     }
 
     scroll() {
-        const winHeight = window.innerHeight;
-        const rect = this.viewer.el.getBoundingClientRect();
-        const progress = Math.min(
-            100,
-            Math.round((Math.abs(rect.top - winHeight) / rect.height) * 100)
+        const progress = Math.round(
+            (window.pageYOffset /
+                (document.body.scrollHeight - window.innerHeight)) *
+                100
         );
```

Comment:
Beware that this is functionally different, and will give inaccurate %s if the page has stuff above or below the incito. (And never reach 100% I think?)

Reply:
@klarstrup That is true but I just opted out of it so the scrollbar matches the percentage.

Reply:
As long as it is intentional :]

Reply:
It is and it's just a demo page so they can always alter it so their needs.

Reply:
But this is not demo page specific :/ it's used by `IncitoPublicationKit` if a `.sgn-incito__progress` element is present within the viewer target - Fakta and Elgiganten and probably several others are using this feature on their own websites right now.

Reply:
Right, you are totally right. I still think it's best like this tbh. Do you feel it's better the other way?